src/modules/displaymanager/main.py

```diff
@@ -327,27 +403,53 @@ def run():
     if "lightdm" in displaymanagers:
         if have_dm("lightdm", root_mount_point):
             if enable_basic_setup:
-                libcalamares.utils.target_env_call(['mkdir', '-p', '/run/lightdm'])
-
-                if libcalamares.utils.target_env_call(['getent', 'group', 'lightdm']) != 0:
-                    libcalamares.utils.target_env_call(['groupadd', '-g', '620', 'lightdm'])
-
-                if libcalamares.utils.target_env_call(['getent', 'passwd', 'lightdm']) != 0:
-                    libcalamares.utils.target_env_call(['useradd', '-c', '"LightDM Display Manager"',
-                                                    '-u', '620', '-g', 'lightdm', '-d', '/var/run/lightdm',
-                                                    '-s', '/usr/bin/nologin', 'lightdm'])
-
-                libcalamares.utils.target_env_call(['passwd', '-l', 'lightdm'])
-                libcalamares.utils.target_env_call(['chown', '-R', 'lightdm:lightdm', '/run/lightdm'])
-                libcalamares.utils.target_env_call(['chmod', '+r' '/etc/lightdm/lightdm.conf'])
+                libcalamares.utils.target_env_call(
+                    'mkdir', '-p', '/run/lightdm'
+                    )
+
+                if libcalamares.utils.target_env_call(
+                        'getent', 'group', 'lightdm'
+                        ) != 0:
+                    libcalamares.utils.target_env_call(
+                        'groupadd', '-g', '620', 'lightdm'
+                        )
+
+                if libcalamares.utils.target_env_call(
+                        'getent', 'passwd', 'lightdm'
+                        ) != 0:
+                    libcalamares.utils.target_env_call(
+                        'useradd', '-c',
+                        '"LightDM Display Manager"',
+                        '-u', '620',
+                        '-g', 'lightdm',
+                        '-d', '/var/run/lightdm',
+                        '-s', '/usr/bin/nologin',
+                        'lightdm'
+                        )
+
+                libcalamares.utils.target_env_call('passwd', '-l', 'lightdm')
+                libcalamares.utils.target_env_call(
+                    'chown', '-R', 'lightdm:lightdm', '/run/lightdm'
+                    )
+                libcalamares.utils.target_env_call(
+                    'chmod', '+r' '/etc/lightdm/lightdm.conf'
+                    )
 
             if default_desktop_environment is not None:
-                os.system("sed -i -e \"s/^.*user-session=.*/user-session={!s}/\" {!s}/etc/lightdm/lightdm.conf".format(
-                          default_desktop_environment.desktop_file, root_mount_point))
+                os.system(
+                    "sed -i -e \"s/^.*user-session=.*/user-session={!s}/\"",
+                    "{!s}/etc/lightdm/lightdm.conf".format(
```

Comment:
Why? First is `user-session={!s}` with `default_desktop_environment.desktop_file` and second is `{!s}/etc/lightdm/lightdm.conf` with `root_mount_point`. What do you consider to be wrong with the first one?